.github/workflows/require-checklist.yml

```diff
@@ -0,0 +1,23 @@
+name: Require Reviewer Checklist
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
```

Comment:
on PR open everything will be unchecked. any worth running then?

Reply:
Makes sense. It would alway start failed. Removing.

Reply:
Thinking about it. A PR could start in the correct state. The author could enter `[x]` in the pull request starting form and not touch it again. For that case, the initial open would be useful. I brought it back